src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      { age && (
+      <p className="Person__age">
+        {'I am '}
+        {age}
+      </p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
```

Comment:
refer to docs and fix this: https://react.dev/learn/rendering-lists
you can also move it to const outside of template (before it)

`const spouse = [your conditional string]`
and then in template use `{spouse}`

but it's obligatory